scripts/setup/flush-memcached

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh -xe
 
 # Flush memcached
-echo 'flush_all' | nc localhost 11211
+echo 'flush_all\nquit' | nc localhost 11211
```

Comment:
What about `echo 'flush_all' | nc -w 0 localhost 11211`? Does it work for you?

I just tried it on a Linux box and seems to be fine and it works on OpenBSD too